### DISCUSSION TITLE
beginner please need help Bryant

### DISCUSSION THREAD

**Discussion by ddave63** [bmoscon/cryptofeed#921] [General]
Hi Bryant,

I would like to use your cryptofeed package and I am quite a beginner and dont really understand.
Can you please help me to provide a Guide line /example please ?
I would like to fetch market data from exchanges and store them in a file.
After that I want to use a backend tool.

Just need your guideline to understand better and have an example to train myself :)

Thanks a lot in advance,
Cheers,

**Comment by bmoscon** [comment]
the docs are here: https://github.com/bmoscon/cryptofeed/tree/master/docs

and there is an example in the readme. You can also find specific examples for various usages in the examples/ directory.

**Comment by ddave63** [comment]
Thanks a lot BryantOn 24 Oct 2022, at 10:56 PM, Bryant Moscon ***@***.***> wrote:﻿
the docs are here: https://github.com/bmoscon/cryptofeed/tree/master/docs
and there is an example in the readme. You can also find specific examples for various usages in the examples/ directory.
